Controllers/AdminController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Controllers;
+
+class AdminController  extends BasicController
+{
+
+  public function __construct()
+  {
+    parent::__construct();
+    $this->title = "Admin page";
+  }
+
+  /**
+   * Callback for /admin route.
+   */
+  public function adminPageAction() {
+    $this->content = $this->render('/views/layouts/admin_content.php');
+    $sidebar = $this->render('/views/layouts/admin_sidebar.php');
```

Comment:
Considering that this template contains a search form, it should be renamed to something that suggests that and moved under /views/forms. For example: admin_search_form.tpl.php
Ideally, any template file should use .tpl.php extension.